src/update_collection_metadata.py

```diff
@@ -199,6 +199,11 @@ async def update_all_collections(
 
     return Completed(message="All new releases have been recorded.")
 
-if __name__ == "__main__":
-   import asyncio
-   asyncio.run(update_all_collections())
+
+# if __name__ == "__main__":
+## ALL COLLECTIONS
+# asyncio.run(update_all_collections())
```

Comment:
Should this be uncommented?

Reply:
I was mostly just trying to make it clear how one could do either locally, I did intentionally comment it out to be explicit that it has no bearing on the automated function of the flow - I can remove and add to the readme if that's better

Reply:
Oh gotcha, makes sense!